docs/sql-guide/functions/function-parsetimestamp.md

```diff
@@ -0,0 +1,138 @@
+---
+title: PARSETIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+## Before you begin
+* [Learn about unix epoch/unix time](https://en.wikipedia.org/wiki/Unix_time){:target="_blank"}
+
+# PARSETIMESTAMP() function
+
+The `PARSETIMESTAMP()` function converts the input time value in string format to timestamp based on the format provided.
+
```

Comment:
Before you begin goes here.

The aim is to give the user any prerequisites and requirements the user needs to have already performed/have that would block the completion of whatever is being described in the body of the page.

So for instance, you can't create a Cloud table without having first added a Cloud database.

Reply:
We dont need that and it was left by mistake. so removed in latest commit.